chapter05/jxp/ch5ex1.pl

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+
+my @lines;
+
+while( my $line = <> ){
+  if ( $line !~ /\n$/  ) {
+    $line .= "\n";
```

Comment:
is this so there's a newline after the final line? you could have put a print "\n"; after the loop